test/fixtures/dialects/bigquery/select_struct.yml

```diff
@@ -327,3 +327,32 @@ file:
               table_reference:
                 naked_identifier: table
 - statement_terminator: ;
+- statement:
```

Comment:
Did you generate this manually? Looks like you need to update the hash at the top of the file.

The easiest way is to generate it automatically: `python test/generate_parse_fixture_yml.py`